cmd/plugin/builder/template/plugintemplates/plugin-tooling.mk.tmpl

```diff
@@ -81,7 +81,7 @@ plugin-build-install-local: plugin-build-local plugin-install-local
 
 .PHONY: plugin-install-local ## Install all plugins from local plugin artifacts directory
 plugin-install-local:
-	tanzu plugin install all --local $(PLUGIN_BINARY_ARTIFACTS_DIR)/$(GOHOSTOS)/$(GOHOSTARCH)
+	tanzu plugin install all --local-source $(PLUGIN_BINARY_ARTIFACTS_DIR)/$(GOHOSTOS)/$(GOHOSTARCH)
```

Comment:
Can we add a comment in the template file specifying that users will need Tanzu CLI v1.0.0 or greater to run this make target?
Just thinking if a user is using an old version of tanzu-cli with a new make file this will not work.

Reply:
Good point.
Will update.